test/helpers/utils_test.go

```diff
@@ -0,0 +1,54 @@
+package helpers
+
```

Comment:
Is this ran during the CI? I searhced for `cilium/test` in the unit tests from the build in this PR but it didn't say anything about unit tests for `test/` being ran.

Reply:
Fixed.